generics/constants/common.js

```diff
@@ -69,5 +69,11 @@ module.exports = {
     "CREATE_SURVEY": "/create-survey/",
     "SUBMISSION_STATUS_COMPLETED" : "completed",
     "EXPIRED": "expired",
-    "STARTED":"started"
+    "STARTED":"started",
+    "AWS_SERVICE": "AWS",
+    "WRITE_PERMISSION": "w",
+    "READ_PERMISSION": "r",
+    "PUBLIC" : "public",
+    "PRIVATE" : "private",
+    "UPLOAD_FILE":"project/cloud-services/elevate-project/v1/files/upload"
```

Comment:
@Prajwal17Tunerlabs is this used any where?

Reply:
it wasnt used anywhere, i removed it